documentation/3_Interoperability.md

```diff
@@ -0,0 +1,375 @@
+of a participant. However, the complexity of achieving interoperability
+across multiple Data Spaces also greatly depends on how those are
+related.
+
+The simplest model is probably a hierarchical direct-dependency of
+Data Spaces. In a larger Data Space a smaller sub-Data Space could be
+created with additional rules, utilizing the governance model of the
+overarching Data Space, but introducing additional policies for the sub
+Data Space. E.g. think of an industrial Data Space where one participant
+wants to share data only with their direct suppliers instead of the
+entire Data Space. This can be realized as a separate Data Space or within
+the larger industrial Data Space by having an additional DGA with
+additional membership policies and specifying additional semantic models
+and processes. In our example one participant could specify that for a
+specific set of data other participants need to prove that they are
+suppliers of this participant and understand a specific semantic models
+and processes provided by this participant. This can be regarded as a
+hierarchical relationship between two Data Spaces. In this case
+interoperability should be straightforward to achieve.
+
+Another model is Data Space peers. Two Data Spaces operating in different
+domains, but with a substantial overlap of participants, which also
+require data from both Data Spaces for many use cases. To reduce the
+burden on participants guaranteeing interoperability, the two Data Spaces
+might agree on same requirements for protocols, semantic models and also
+on organizational processes, including agreements on legal equivalency.
+
+Last but not least, there is the case of completely unrelated Data Spaces
+where one or just a few participants have the need to access data from
+multiple Data Spaces. In this case the burden of interoperability fully
+lies on the participants as they will need to be able to comply with
+processes in both Data Spaces and potentially will need to provide
+completely separate technical environments to access data in different
+Data Spaces.
+
+No matter how Data Spaces are related and cross-Data Space
+interoperability will be achieved, it is always going to be the
+responsibility of the participant to keep track of which data was
+acquired through which Data Space and what obligations came with it.
+Especially if use cases need to combine data from different Data Spaces
+sophisticated data management will be required.
+
+## Trust Frameworks and Trust Anchors
+
+As the DGA also defines which Trust Frameworks and which Trust Anchors
+can be used by participants within a Data Space all the before mentioned
+interoperability facets also apply to Trust Frameworks and Trust
+Anchors. As it is very likely that Trust Frameworks and Trust Anchors
+will support multiple Data Spaces it is especially important that the
+applicable protocol versions and semantic models are clearly defined.
+
+## Improving Interoperability
```

Comment:
This section could be improved. It is pretty short and the message is not really clear.

What would be a preferred way to improve interoperability? Could we make this something like an adoption model, e.g 
- phase 1: agreements between 2 participants
- phase 2: agreements in a group of participants, like a hierarchical ds
- phase 3: Agreements within a common trust framework
- phase 4: agreements between service providers, like trust frameworks
- phase 5: general agreements in a ds in the dsga governance framework
- phase 6: agreements between different dsga --> federation?

This would mean a shift from responsibility of the participant to the governance bodies. 

Finally, a phase could be the provisioning of standards, regulations or trade agreements?

Reply:
yes, this would be a great addition. Shall we work out this paragraph in the WG meeting?

Reply:
Gave it a start in #62

Reply:
like the change, made a small additional change in the wording for service providers